packages/react/carbon.yml

```diff
@@ -445,6 +460,20 @@ assets:
         name: Storybook
         action: link
         url: https://react.carbondesignsystem.com/?path=/docs/components-search
+    section:
+    name: Section
+    framework: react
+    status: stable
+    type: component
+    platform: web
+    thumbnailPath: ./thumbnails/section.svg
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://react.carbondesignsystem.com/?path=/docs/heading
```

Comment:
Is this URL right? A "heading" path, not "section" path?

Reply:
Yeah this is correct. Section doesn't have it's own story, it's baked into the default story named `heading`